spec/index.bs

```diff
@@ -67,6 +68,11 @@ spec: service-workers; urlPrefix: https://slightlyoff.github.io/ServiceWorker/sp
         text: ServiceWorkerGlobalScope; url: service-worker-global-scope-interface
         text: ServiceWorkerRegistration; url: service-worker-registration-interface
 
+spec: notifications; urlPrefix: https://notifications.spec.whatwg.org/#
```

Comment:
I'm not quite sure how to do so in bikeshed, but this should probably somehow be a non-normative reference. Now it looks like this spec depends on notifications, while it's only examples that do so...